text-show-instances.cabal

```
@@ -209,6 +214,7 @@ test-suite spec
                        Spec.Data.ScientificSpec
                        Spec.Data.ShortTextSpec
                        Spec.Data.VectorSpec
+                       Spec.Data.VectorNonEmptySpec
```

Style nit pick: put this module in alphabetical order:

```suggestion
                       Spec.Data.VectorNonEmptySpec
                       Spec.Data.VectorSpec
```